### DISCUSSION TITLE
Bayesian Optimization to find best transmitter azimuth

### DISCUSSION THREAD

**Discussion by manoj-kumar-joshi** [NVlabs/sionna#534] [Show and tell]
As we know that Gradient descent algorithm may not produce best results, when we Optimize Transmitter Orientation i(For more than one). This notebook implements the cool Bayesian Optimization 
https://github.com/manoj-kumar-joshi/Sionna_BayesianOptimization/tree/main
Below are some screenshots of output ...
![BO_output](https://github.com/user-attachments/assets/1223bf6e-4e6b-4f5f-863a-8d8859eb8518)

**Comment by jhoydis** [comment]
Very nice! Thanks for sharing.
